api/schemas/user.py

```diff
@@ -1,17 +1,60 @@
 import graphene
-from graphene import relay
-from graphene_sqlalchemy import SQLAlchemyObjectType
+from flask_graphql_auth import create_refresh_token
+from graphene import relay, Mutation
+from graphene_sqlalchemy import SQLAlchemyObjectType, SQLAlchemyConnectionField
 
-from models import Users as UserModel
+from flask_bcrypt import Bcrypt
+
+from flask import current_app as app
+
+from ..models import Users as UserModel
+
+from ..db import db_session
+
+from flask_graphql_auth import (
+	AuthInfoField,
+	GraphQLAuth,
+	get_jwt_identity,
+	get_raw_jwt,
+	create_access_token,
+	create_refresh_token,
+	query_jwt_required,
+	mutation_jwt_refresh_token_required,
+	mutation_jwt_required,
+)
 
 
 class User(SQLAlchemyObjectType):
 	class Meta:
```

Comment:
Class should have to following exclude_fields = 'user_password', this ensures that no query from outside can access the passwords columns